docs/source/release_notes/release-notes.rst

```diff
@@ -42,47 +42,46 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
+         - The ``DebugLogger`` class was moved from package ``com.palantir.timestamp`` in project ``timestamp-impl`` to ``com.palantir.util`` in project ``atlasdb-commons``.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1445>`__)
+
     *    - |improved|
          - Increase default Cassandra pool size from minimum of 20 and maximum of 5x the minimum (100 if minimum not modified) connections to minimum of 30 and maximum of 100 connections.
-           This allows for better handling of bursts of requests that would otherwise require creating many new connections to Cassandra from the clients.
+           This has empirically shown better handling of bursts of requests that would otherwise require creating many new connections to Cassandra from the clients.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1402>`__)
 
     *    - |new|
          - Added metrics to SnapshotTransaction to monitor durations of various operations such as ``get``, ``getRows``, ``commit``, etc.
-           Atlas users should use ``AtlasDbMetrics.setMetricRegistry`` to set a ``MetricRegistry``.
+           AtlasDB users should use ``AtlasDbMetrics.setMetricRegistry`` to set a ``MetricRegistry``.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1429>`__)
 
     *    - |new|
+         - Added metrics in Cassandra clients to record connection pool statistics and exception rates.
+           These metrics use the global ``AtlasDbRegistry`` metrics.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1380>`__)
+
+    *    - |new|
          - There is now a ``TimestampMigrationService`` with the ``fast-forward`` method that can be used to migrate between timestamp services.
```

Comment:
I am not very sure if this should be flagged in this release but as backups are already in the field I thought this will be useful as we have extended ``fast-forward`` to all timestamp services.